collector/collector.cpp

```diff
@@ -40,7 +40,7 @@ extern "C" {
 #include "LogLevel.h"
 #include "Logging.h"
 #include "Utility.h"
-#include "Version.h"
+#include "collector/Version.h"
```

Comment:
Why not rename the file to `CollectorVersion.h`? It would make it consistent with other files we have and it feels off that we only specify the include directory for _this specific file_.

Reply:
Yup, directory is clunky, done.